docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.8"
+name: irma_email_issuer
+
+services:
+  # Irma issuer service
+  irmaserver:
+    image: ghcr.io/privacybydesign/irma:v0.13.2
+    container_name: irma_server
+    working_dir: /irmago
+    ports:
+      - 8088:8088
+    expose:
+      - 8088
+    networks:
+      - irma-net
+    entrypoint:
+      - "irma"
+      - "server"
+      - "--no-auth=false"
+      - "--requestors={\"irma_email_issuer\":{\"auth_method\":\"publickey\",\"key_file\": \"/config/pk.pem\"} }"
+      - "--port=8088"
+      - "--jwt-privkey-file=/config/sk.pem"
+      - "--url=http://ip-address:8088" # Your localhost IP-address here. This is required for the app to be able to connect to the issuer
```

Comment:
You can also do `- "--url=http://${IP}:8088"` and then run `IP="192.168.1.1" docker-compose up`. Of course that introduces some difference with the SMS issuer, so that's a bit of a drawback.